Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -102,21 +103,36 @@ open class BpkTextField @JvmOverloads constructor(
     }
 
     this.iconTintColor = iconColor
-    setTextColor(textColor)
+    setTextColor(
+      colorStateList(
+        color = textColor,
+        pressedColor = textColor,
+        focusedColor = textColor,
+        activatedColor = textColor,
+        disabledColor = textColorDisabled,
+      )
+    )
     setHintTextColor(
-      ColorStateList(
-        arrayOf(intArrayOf(android.R.attr.state_focused), intArrayOf()),
-        intArrayOf(hintFocusedColor, hintNormalColor)
+      colorStateList(
+        color = hintNormalColor,
+        pressedColor = hintNormalColor,
+        focusedColor = hintFocusedColor,
+        activatedColor = hintFocusedColor,
+        disabledColor = textColorDisabled,
       )
     )
 
-    val padding = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
+    val paddingHorizontal = resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
+    val paddingVertical = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
+      resources.getDimensionPixelSize(R.dimen.bpkSpacingSm)
+    setPaddingRelative(paddingHorizontal, paddingVertical, paddingHorizontal, paddingVertical)
+    compoundDrawablePadding = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
       resources.getDimensionPixelSize(R.dimen.bpkSpacingSm)
-    setPaddingRelative(padding, padding, padding, padding)
-    compoundDrawablePadding = padding
 
     gravity = Gravity.START or Gravity.CENTER_VERTICAL
     this.background = background
+    this.minHeight = resources.getDimensionPixelSize(R.dimen.bpkSpacingXxl) +
```

Comment:
We still want to match the design in a single-line version, therefore we set minHeight to 48dp and center the text vertically. Since text and field have fixed sizes, vert padding should be corrent.